generators/php/math.js

```diff
@@ -283,40 +277,40 @@ Blockly.PHP['math_on_list'] = function(block) {
       // Mode of [3, 'x', 'x', 1, 1, 2, '3'] -> ['x', 1].
       var functionName = Blockly.PHP.provideFunction_(
           'math_modes',
-          [ 'function ' + Blockly.PHP.FUNCTION_NAME_PLACEHOLDER_ +
+          ['function ' + Blockly.PHP.FUNCTION_NAME_PLACEHOLDER_ +
               '($values) {',
-            '  $v = array_count_values($values);',
-            '  arsort($v);',
-            '  foreach($v as $k => $v){$total = $k; break;}',
-            '  return array($total);',
-            '}']);
+           '  $v = array_count_values($values);',
+           '  arsort($v);',
+           '  foreach($v as $k => $v){$total = $k; break;}',
```

Comment:
This lines does need curly braces because there are two statements, however as discussed, there are problems with this implementation and appropriate unit tests are being added as well as a fix.